### PR TITLE
Fixes #46 - Create share intent in IC4DNewsDetailActivity

### LINKED ISSUE #46: Create share intent in IC4DNewsDetailActivity

### PR DESCRIPTION
**Pull request by noahalorwu**

### CODE REVIEW

File: ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/screens/news/detail/ICT4DNewsDetailActivity.kt [hunk 1]
```diff
@@ -25,10 +28,12 @@ class ICT4DNewsDetailActivity : BaseActivity<ICT4DNewsDetailViewModel, ActivityI
...
         super.onCreate(savedInstanceState)
-
         fab.setOnClickListener { view ->
-            Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
-                .setAction("Action", null).show()
+            val linkShare = intent.getParcelableExtra<News>(KEY_NEWS_LIST_MODEL)
+            val sharingIntent = Intent(Intent.ACTION_SEND)
+            sharingIntent.putExtra(Intent.EXTRA_TEXT, linkShare.link)
+            sharingIntent.setType("text/plain")
+            this.startActivity(Intent.createChooser(sharingIntent, resources.getText(R.string.send_to)))
```
Comment (spipau) [inline ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/screens/news/detail/ICT4DNewsDetailActivity.kt]: We use ANKO in our project which makes programming Android in Koltin much easier, you should check it out:
https://github.com/Kotlin/anko/wiki/Anko-Commons-%E2%80%93-Intents

The provide Intent Extensions you can replace you code simply with: `share(text, [subject])`, pretty cool 😉 
Please use it!

Please also replace the String with:

```
[Title of the Blog Post] by [Author] - [date]
[link to the blog post]```

**Comment by spipau** [comment]
Cool, I will check your changes as soon as I have time for it 👍 
Small hint: if the Travis CI builds fail because of Ktlint, then run the Ktlint Gradle task locally (Gradle Tab to the right) and it will tell you why and where to fix it!

**Comment by noahalorwu** [comment]
Thanks for taking a look at my codes and improving them. I have taken note of the above mention things i should be careful of. 👍:zap: :green_heart: :twisted_rightwards_arrows:

**Comment by spipau** [comment]
Cool thing, I will write the others soon a message... I have good news 😉

**Pull request merged**